api/types/schedule.go

```diff
@@ -336,6 +350,19 @@ func (s *Schedule) SetError(err string) {
 	s.Error = &err
 }
 
+// SetNextRun sets the NextRun field.
+//
+// When the provided Schedule type is nil, it
+// will set nothing and immediately return.
+func (s *Schedule) SetNextRun(err int64) {
```

Comment:
confused about the var name `err int64` is it an error?

Reply:
Good catch!